tests/TypeReconciliation/RedundantConditionTest.php

```diff
@@ -1546,6 +1546,27 @@ function f(array $p) : void {
                     }',
                 'error_message' => 'RedundantCondition',
             ],
+            'https://github.com/vimeo/psalm/issues/8932' => [
```

Comment:
Having a reference is a great idea but I still would like to have some kind of title Maybe a comment in the code itself? If someday, we were to migrate from github, we could loose the meaning behind the tests completely